Corona-Warn-App/src/main/res/values-de/release_info_strings.xml

```diff
@@ -17,22 +17,18 @@
 
     <!-- XHED: Titles for the release info screen bullet points -->
     <string-array name="new_release_title">
-        <item>Link auf RKI-Umfrage</item>
-        <item>Einwilligung zur Datenspende (optional)</item>
-        <item>Anpassungen der Risiko-Karten</item>
-        <item>Fortführung der Risiko-Ermittlung nach Teilung der Zufalls-IDs</item>
-        <item>Einführung von Registerkarten</item>
-        <item>Mehr Informationen zum Testablauf</item>
+        <item>Zusatzfunktionen im Kontakt-Tagebuch</item>
```

Comment:
Should we add "" to the new strings e.g. "Zusatzfunktionen im Kontakt-Tagebuch" to be aligned with the other strings?

Reply:
So far, we did not do this in this file at all.

Reply:
alright, should be fine then :)

Reply:
This was introduced by the translation tool, don't ask me why...